src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -0,0 +1,352 @@
+import json
+import errno
+import cephfs
+import logging
+import orchestrator
+
+from subvolspec import SubvolumeSpec
+from subvolume import SubVolume
+from exception import VolumeException
```

Comment:
s/exception/.exception/

Reply:
doesn't python look for the module in the current directory first? is there a need to explicitly mention with `.`?

Reply:
If you want to do relative imports, as you intend to do here, you need to precede the module with a '.' . See other examples,
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/ansible/module.py#L19

If you want to do absolute imports, then you need to use the full path of the module from project's root folder.
https://realpython.com/absolute-vs-relative-python-imports/